api/prisma/schema.prisma

```diff
@@ -424,6 +425,19 @@ model ListingMultiselectQuestions {
   @@map("listing_multiselect_questions")
 }
 
+model ApplicationLotteryPositions {
+  ordinal               Int
+  listingId             String                @map("listing_id") @db.Uuid
+  listings              Listings              @relation(fields: [listingId], references: [id], onDelete: Cascade, onUpdate: NoAction)
```

Comment:
General question: why is this pluralized? Is it returning a list of just one listing?

Reply:
yeaaa thats sort of a hold over/problem we have throughout our schema. I think it would be better to leave this as plural to match up with other similar models but we do have a backend TODO to clean up this plural/singular issues